src/Game.js

```diff
@@ -0,0 +1,51 @@
+import Player from '../src/Player';
+import Round from '../src/Round';
+
+class Game {
+  constructor(data) {
+    //   constructor(response_code, categories, clues ) {
+    // Object.assign(this, { response_code, categories, clues });
+    this.categories = data.categories
+    this.clues = data.clues;
+    this.players = [];
+    this.roundCounter = 0; // goes up to 3
+    this.fourCategories = [];
+    this.round;
+  }
+
+  getFourCategories() { 
```

Comment:
Currently this function is set tot modify the this.forCategories array in the constructor. Since this is going to be called each round, would it be possible to set the function to return the array? Then we wouldn't need a constructor variable and it would output the array where the function is called.

Reply:
@foster55f is going to update once merged